README.md

```diff
@@ -74,6 +74,29 @@ in Python, Julia, R, Rust, and C.
 
 Examples of other functionality can be found in the `test` folder for each interface.
 
+## Software Using BridgeStan
```

Comment:
I like "Software using BridgeStan".  That's technically sentence capitalization rather than title capitalization.  But it's OK to do this either way as long as it's consistent.